pkg/asset/installconfig/pullsecret.go

```diff
@@ -24,7 +24,7 @@ func (a *pullSecret) Generate(asset.Parents) error {
 		{
 			Prompt: &survey.Password{
 				Message: "Pull Secret",
-				Help:    "The container registry pull secret for this cluster, as a single line of JSON (e.g. {\"auths\": {...}}).\n\nYou can get this secret from https://cloud.openshift.com/clusters/install#pull-secret",
+				Help:    "The container registry pull secret for this cluster, as a single line of JSON (e.g. {\"auths\": {...}}).\n\nYou can get this secret from https://cloud.openshift.com/openshift/install/pull-secret",
```

Comment:
Should be cloud.redhat.com now, no?

Reply:
ugh, of course. Good eye Trevor, thank you. me~~
fixed in ae44ae1